components/ListingsStarshipCard/ListingsStarshipCard.js

```diff
@@ -0,0 +1,79 @@
+import useSWR from "swr";
+import Image from "next/image";
+import styled from "styled-components";
+import Flex from "../Layout/Flex";
+
+const StyledImageWrapper = styled.div`
+  height: 84px;
+  width: 84px;
+  border-radius: 50%;
+  overflow: hidden;
+  position: relative;
+`;
+
+const StyledCard = styled.div`
+  border-radius: 11px;
+  border: 0.5px solid var(--secondary-color);
+  padding: 1rem;
+  background-color: var(--button-color));
+`;
+
+const StyledStarshipImage = styled(Image)`
+  object-fit: cover;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  margin: auto;
+`;
+
+const StyledParagraph = styled.p`
+  color: var(--secondary-color);
+`;
+const StyledParagraphPreis = styled.p`
+  color: var(--secondary-color);
+  font-size: 2rem;
+`;
+
+const StyledBuyButton = styled.button`
+  background-color: #000000;
+  border-radius: 5px;
+  border: 0.3px solid var(--secondary-color);
+  height: 2rem;
+  width: 6.25rem;
+  color: var(--secondary-color);
+`;
+
+export default function ListingsStarshipCard({ userId, name, img, preis }) {
+  const { data: users, isLoading } = useSWR("/api/users");
+  if (!users || isLoading) {
+    return <h1>Loading...</h1>;
+  }
+  const mainUser = users[0];
+
+  if (userId === mainUser._id) {
+    return;
+  }
+  return (
+    <StyledCard>
+      <Flex justifyContent="space-around">
+        <Flex direction="column" alignItems="center" gap=".5px">
+          <StyledImageWrapper>
+            <StyledStarshipImage
+              src={img}
+              alt={name}
+              height={0}
+              width={0}
+              layout="responsive"
+            />
+          </StyledImageWrapper>
+          <StyledParagraph>{name}</StyledParagraph>
+        </Flex>
+        <Flex direction="column" alignItems="center" gap="5px">
+          <StyledParagraph>Price: </StyledParagraph>
+          <StyledParagraphPreis>{preis}</StyledParagraphPreis>
```

Comment:
I know from a different PR that you are planning on renaming "preis" to "price". Here you named the component `StyledParagraphPreis`. To avoid having too much changes in the future, I would maybe suggest to change the naming now so you don't have to rename so many things at once! :)